lighthouse-core/test/report/html/renderer/report-renderer-test.js

```diff
@@ -278,14 +278,17 @@ describe('ReportRenderer', () => {
     });
 
     it('renders without axe violations', (done) => {
-      const container = renderer._dom._document.body;
+      const container = renderer._dom._document.createElement('main');
       const output = renderer.renderReport(sampleResults, container);
+      renderer._dom._document.body.appendChild(container);
 
       const config = {
         rules: {
           // Reports may have duplicate ids
           // https://github.com/GoogleChrome/lighthouse/issues/9432
           'duplicate-id': {enabled: false},
+          'duplicate-id-aria': {enabled: false},
+          'landmark-no-duplicate-contentinfo': {enabled: false},
```

Comment:
I'm not completely certain whether it's fair to associate _both_ of these disabled tests with #9432.  `duplicate-id-aria` seems likely to be related; I'm not 100% sure on whether `landmark-no-duplicate-contentinfo` is.